### ISSUE TITLE
rf_count frequencies folder

### ISSUE THREAD

**Issue by dfporter** [dincarnato/RNAFramework#43]
rf_count produces a subfolder called frequencies with a single file containing each dinucleotide pair, including AA/TT/CC/GG, and a float value, with the floats summing to 1. The perl that makes this is opaque to me, what is the interpretation of this file? Does "AA" represent a deletion of an "A" base?

**Comment by dincarnato** [comment]
Dear Douglas,

you are right. This should be added to the manual.
The `frequencies/` folder is generated when rf-count is ran with the `-of` parameter. It contains the frequency of each base substitution in the experiment. The AA/CC/GG/TT should all be 0, cause it's like saying that the base was substituted by itself.

__Important:__ we noticed over the past 2 weeks that there was a bug causing those frequencies to be miscalculated in the presence of indels. The bug is now fixed and we have already pushed the working version of rf-count, so i would recommend you to do a `git pull` and rerun your analysis. We will release the next official version of RNAFramework soon.
